doc/source/changelog.rst

```diff
@@ -1,6 +1,35 @@
 Changelog
 ******************************
 
+- **v0.5** (*2017-03-01*):
+
+  - New
+
+    - Add new looper version tracking, with `--version` option and printing version at runtime
+
+    - Add support for asterisks in file paths
+
+    - Add support for multiple pipeline directories in priority order
+
+    - Revamp of messages make more intuitive output
+
+    - Colorize output
+
+    - Adds new internal object structure for handling ProtocolMapping and PipelineInterface
```

Comment:
I added them just because this is both looper and models; those are publicly relevant for models users (not for looper users)... but I will take them out since this is looper after all.